pegasus/sites.v3/code.org/public/css/mobile-footer.css

```diff
@@ -0,0 +1,155 @@
+#footer-links {
+  float: right;
+  color: white;
+  font-size: 12px;
+  z-index: 2;
+}
+
+#footer-links ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+#footer-links ul li {
+  display: inline;
+  text-indent: 0;
+  padding: 0;
+}
+
+#footer-links ul li:before {
+  content: none;    /* Needed to remove the bullet disc. This was set from other CSS. */
+}
+
+#footer-links ul li:after {
+  content: "|";     /* Vertical separator between the links */
+  margin: 0 12px;
+  opacity: 0.5;
+}
+
+#footer-links ul li:last-child:after {
+  content: none;    /* No separator needed for the last item */
+}
+
+#footer-links a {
+  color: white;
+  font-weight: normal;
+  font-family: 'Gotham 4r', sans-serif;
+  opacity: 0.5;
+  text-decoration: none;
+}
+
+#footer-links a:hover {
+  opacity: 1.0;
+}
+
+#footer-toggle {
+  display: none;
+  text-align: center;
+  border-bottom: 1px solid #aaa2bd;
+  padding: 8px 16px;
+  margin: 0 24px;
+  cursor: pointer;
+  margin-bottom: 10px;
+}
+
+#copyright {
+  float: left;
+  color: white;
+  font-size: 12px;
+  opacity: 0.5;
+}
+
+.social-media a {
+  text-decoration: none;
+  color: white;
+}
+
+/*medium screens*/
+@media screen and (max-width: 970px)
```

Comment:
I did it this way as an attempt minimize redundancy in the css. For example, there is styling in the medium screen size that is also applicable to the small screen size. They are not mutually exclusive.